lib/spack/spack/environment/shell.py

```diff
@@ -68,7 +68,17 @@ def activate_header(env, shell, prompt=None, view: Optional[str] = None):
             cmds += "    fi;\n"
             cmds += '    export SPACK_OLD_PS1="${PS1}";\n'
             cmds += "fi;\n"
-            cmds += 'export PS1="%s ${PS1}";\n' % prompt
+            cmds += 'if [ -n "${TERM:-}" ] && [ "${TERM#*color}" != "${TERM}" ] && \\'
```

Comment:
```suggestion
            cmds += 'if [ -n "${TERM:-}" ] && [ "${TERM#*color}" != "${TERM}" ] && '
```

Reply:
Replace `cmds +=` with `f""" ... """` to guard against those issues?

Reply:
Yeah I like it better.  Done, with `textwrap.dedent()` for readability.

Reply:
of course python doesn't do that by default, lol.